NEWS.md

```diff
@@ -1,6 +1,7 @@
 ## 25.0.0 IN-PROGRESS
 
 * Fixed effective location migration script for holdings records (MODINVSTOR-940)
+* Fixing mg bugfest and clean deployment schema difference ([MODINVSTOR-935] (https://issues.folio.org/browse/MODINVSTOR-935))
```

Comment:
I would probably change this in a similar way to the feedback I gave on the back port pull request